client/src/components/Modal/Modal.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components/macro";
+import React from "react";
+import img from "../../assets/hubble-eyes.jpg";
+import PropTypes from "prop-types";
+
+const ModalContainer = styled.div`
+  width: 600px;
+  background-color: #fff;
+  color: black;
+  padding: 0.8rem;
+  border-radius: 30px;
+  box-shadow: 0px 0px 15px 3px #fff;
+
+  img {
+    max-width: 100%;
+    border-radius: 25px;
+  }
+  button {
+    border: none;
+    background-color: #fff;
+    width: 20px;
+    height: 20px;
+    cursor: pointer;
+    text-align: center;
+  }
+`;
+
+const Modal = ({ modalOpen, setModalOpen }) => {
```

Comment:
This is the only thing I'm not sure about...giving the useState through that component...but besides that, love it!:)

Reply:
Thanks for review.. Maybe in a later step, I might outsource the state into am custom hook.